wiki/misc/mappings/en.md

```diff
@@ -0,0 +1,41 @@
+# Customizing the Mappings in Your Development Environment
+
+## What are mappings?
+
+Before a new Minecraft version jar is published to Mojang's servers, it goes through a process called *obfuscation*, where the human-readable class, field and method names are simplified to just a few letters, mainly to optimize the file size, and in other contexts to prevent people from easily understanding your code. Obfuscation makes code very difficult to understand, because most names are just converted to random letters, so this is where mappings come into play.
```

Comment:
There are two sentences talking about making code hard to understand. I would remove the clause from the first sentence and add a `In addition` or something to the second sentence. Probably also split up the last sentence. 

```suggestion
Before a new Minecraft version jar is published to Mojang's servers, it goes through a process called *obfuscation*, where the human-readable class, field and method names are simplified to just a few letters, mainly to optimize the file size. In addition, obfuscation makes code very difficult to understand, because most names are just converted to random letters. This is where mappings come into play.
```